src/common/data/utils.js

```diff
@@ -3,3 +3,21 @@ import get from 'lodash/get';
 export const getLocalizedFieldValue = (field, locale) => {
   return get(field, locale, null);
 };
+
+export const getLocalizedFieldValueWithFallback = (field, locale, fallbackLocales = []) => {
+  const localeValue = getLocalizedFieldValue(field, locale);
+
+  if (localeValue) {
+    return localeValue;
+  }
+
+  let fallbackValue = null;
```

Comment:
You should not be making side-effects (assigning fallbackValue) in `find`, a `forEach` would be more suitable for that kind of stuff. That said, `find` is the correct method you want to be using here, but in a bit different manner. From [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find): 

> The find() method returns the value of the first element in the array that satisfies the provided testing function. Otherwise undefined is returned.

So in this case you should return `true` in the callback for `find` when the value is something that you want to use, or `false` when it's an undesired value. In this case `getLocalizedFieldValue` returns `null` when a value is not found, so in the callback for `find` you should check that whatever `getLocalizedFieldValue` returns is not null e.g. `return getLocalizedFieldValue(field, value) !== null`.

`getLocalizedFieldValue` takes two arguments, `field` and `locale` so you should not be passing any more arguments than necessary to it (remove the `null` argument).

Since `find` returns the first value that we can use, it means that you can assign the return value of `fallbackLocales.find(...)` directly to `fallbackValue`, `const fallbackValue = fallbackLocales.find(...)`